Plugins/Flow.Launcher.Plugin.BrowserBookmark/Languages/en.xaml

```diff
@@ -8,7 +8,7 @@
     <system:String x:Key="flowlauncher_plugin_browserbookmark_plugin_description">Search your browser bookmarks</system:String>
 
     <!--  Settings  -->
-    <system:String x:Key="BookmarkDataSetting">Bookmmark Data</system:String>
+    <system:String x:Key="BookmarkDataSetting">Bookmark Data</system:String>
```

Comment:
however this name does not conform with others, may have a slight chance to cause clash. let's use this pr to fix